CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+Thank you for considering to contribute to Surfbird, whether it is a simple bug report or an entire feature you might have planned out, everything is appreciated and welcome!
+
+### I have a problem/question!
+
+Do you have a question regarding Surfbird or a specific problem with something? if there is no issue for it yet, [please open one](https://github.com/surfbirdapp/surfbird/issues/new)!
+
+### I found a bug!
+
+If you think you found a bug in Surfbird, please search through [all issues](https://github.com/surfbirdapp/surfbird/issues?q=) before opening another one!
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/surfbirdapp/surfbird/issues/new). Be sure to include a title and clear description, as much relevant information as possible, maybe even include a screenshot, the more the better!
```

Comment:
I'd definitely recommend adding an [issue template](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) that people can work off 📝

Reply:
Good idea, I'll add both an issue and a PR template next!